tests/unit-tests/class-llms-blocks-test-blocks.php

```diff
@@ -2,10 +2,11 @@
 /**
  * Test main plugin file.
  *
- * @package  LifterLMS_Blocks/Tests
+ * @package LifterLMS_Blocks/Tests
  *
  * @since 1.5.1
- * @version 1.6.0
+ * @since 1.6.0 Update `test_add_block_category` test to accommodate form fields cat.
+ * @version [version]
```

Comment:
It looks like this wasn't actually updated in this update, right?

```suggestion
 * @version 1.6.0
```

Reply:
right, I'm rebasing this to the master and I'll remove this too. thanks!